eng/update-dependencies/Options.cs

```diff
@@ -22,61 +21,29 @@ public class Options
-                syntax.DefineOptionList(
-                    "product-version",
-                    ref productVersions,
-                    "Product versions to update (<product-name>=<version>)");
-                ProductVersions = productVersions
-                    .Select(pair => pair.Split(new char[] { '=' }, 2))
-                    .ToDictionary(split => split[0].ToLower(), split => split[1]);
-
-                string versionSourceName = null;
-                syntax.DefineOption(
-                    "version-source-name",
-                    ref versionSourceName,
-                    "The name of the source from which the version information was acquired.");
-                VersionSourceName = versionSourceName;
-
-                string gitHubEmail = null;
-                syntax.DefineOption(
-                    "email",
-                    ref gitHubEmail,
-                    "GitHub email used to make PR (if not specified, a PR will not be created)");
-                GitHubEmail = gitHubEmail;
-
-                string gitHubPassword = null;
-                syntax.DefineOption(
-                    "password",
-                    ref gitHubPassword,
-                    "GitHub password used to make PR (if not specified, a PR will not be created)");
-                GitHubPassword = gitHubPassword;
-
-                string gitHubUser = null;
-                syntax.DefineOption(
-                    "user",
-                    ref gitHubUser,
-                    "GitHub user used to make PR (if not specified, a PR will not be created)");
-                GitHubUser = gitHubUser;
-
-                bool computeChecksums = false;
-                syntax.DefineOption(
-                    "compute-shas",
-                    ref computeChecksums,
-                    "Compute the checksum if a published checksum cannot be found");
-                ComputeChecksums = computeChecksums;
-
-                string dockerfileVersion = null;
-                syntax.DefineParameter(
-                    "dockerfile-version",
-                    ref dockerfileVersion,
-                    "Version to the Dockerfiles to update");
-                DockerfileVersion = dockerfileVersion;
-            });
+            DockerfileVersion = dockerfileVersion;
+            ProductVersions = productVersion
+                .Select(pair => pair.Split(new char[] { '=' }, 2))
+                .ToDictionary(split => split[0].ToLower(), split => split[1]);
+            VersionSourceName = versionSourceName;
+            GitHubEmail = email;
+            GitHubPassword = password;
+            GitHubUser = user;
+            ComputeChecksums = computeShas;
         }
+
+        public static IEnumerable<Symbol> GetCliSymbols() =>
+            new Symbol[]
+            {
+                new Argument<string>("dockerfile-version", "Version to the Dockerfiles to update"),
```

Comment:
I realize this was the previous wording...
```suggestion
                new Argument<string>("dockerfile-version", "Version of the Dockerfiles to update"),
```